Subject: [PATCH] shooting: new unreleased matlab that might work, hight is 109
---
Index: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/consts/ShooterCoefficients.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/consts/ShooterCoefficients.java b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/consts/ShooterCoefficients.java
--- a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/consts/ShooterCoefficients.java	(revision 7afd2f9a9e08107475376ee4776ec24533f9d565)
+++ b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/consts/ShooterCoefficients.java	(date 1773680373574)
@@ -5,13 +5,19 @@
 @Config
 public class ShooterCoefficients {
     public static double[] RPM_INTERPOLATION = {400, 0};
-    public static double[] VELOCITY_INTERPOLATION = {1.1413, 3.6006};
+    //public static double[] VELOCITY_INTERPOLATION = {1.1413, 3.6006};
+    public static double[] VELOCITY_INTERPOLATION = {1.1389, 3.6028};
     public static double[] CLOSE_VEL_COEFFS = {415, 100};
     public static double[] FAR_VEL_COEFFS = {415, 100};
     public static final double DISTANCE_THRESHOLD_METERS = 2.5;
-    public static final double[] HOOD_COEFFS = {-0.00842119, 0.12090451, -0.67832152, 1.87398840, -2.66079694, 2.38229058};
-    public static final double[] MIN_VEL_COEFFS = {-0.00753753, 0.14158881, -1.01421641, 3.47785831, -4.57624623, 6.68692950};
-    public static final double[] MAX_VEL_COEFFS = {0.01871128, -0.26276313, 1.42527417, -3.70961862, 5.72298206, 1.59238723};
+
+    //public static final double[] HOOD_COEFFS = {-0.00842119, 0.12090451, -0.67832152, 1.87398840, -2.66079694, 2.38229058};
+    //public static final double[] MIN_VEL_COEFFS = {-0.00753753, 0.14158881, -1.01421641, 3.47785831, -4.57624623, 6.68692950};
+    //public static final double[] MAX_VEL_COEFFS = {0.01871128, -0.26276313, 1.42527417, -3.70961862, 5.72298206, 1.59238723};
+
+    private static final double[] HOOD_COEFFS = {-0.01239214, 0.18153442, -1.03930662, 2.91532883, -4.09402453, 3.07717266};
+    private static final double[] MIN_VEL_COEFFS = {0.03471542, -0.46507830, 2.34789847, -5.43741851, 6.57391478, 1.70533916};
+    private static final double[] MAX_VEL_COEFFS = {0.00506945, -0.08700410, 0.57604875, -1.82881481, 3.88886857, 2.17979845};
 
     public static final double[] HOOD_COEFFS_CLOSE = {-0.00842119, 0.12090451, -0.67832152, 1.87398840, -2.66079694, 2.38229058};
     public static final double[] MAX_VEL_COEFFS_CLOSE = {0.01871128, -0.26276313, 1.42527417, -3.70961862, 5.72298206, 1.59238723};
